File: src/luogu/p5720/Main.java
// https://www.luogu.com.cn/problem/P5720
package luogu.p5720;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int i = sc.nextInt();
        int c = 1;
        while (i > 1) {
            i /= 2;
            c++;
        }
        System.out.println(c);
    }
}

File: src/luogu/p5720/Main.go
// https://www.luogu.com.cn/problem/P5720
package main

import "fmt"

func main() {
	var a int
	fmt.Scanf("%d", &a)
	i := 1
	for a > 1 {
		a /= 2
		i++
	}
	fmt.Println(i)
}
